lib/ansible/plugins/doc_fragments/default_callback.py

```diff
@@ -18,9 +18,9 @@ class ModuleDocFragment(object):
         env:
           - name: DISPLAY_SKIPPED_HOSTS
             deprecated:
-              why: environment variables without "ANSIBLE_" prefix are deprecated
+              why: environment variables without C(ANSIBLE_) prefix are deprecated
```

Comment:
Should the quote fixes be done in a separate PR so they can be independently backported?

Reply:
Ah, I see you've already opened a PR for that.

Reply:
Sorry, didn't mention it here, but I created a separate PR (#76698). I've now rebased after that one was merged.